src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -63,6 +63,22 @@ public ObservableCollection<DynamoDropDownItem> Items
             }
         }
 
+        public override NodeInputData InputData()
+        {
+
+            return new NodeInputData()
+            {
+                Id = this.GUID.ToString("N"),
+                Name = this.Name,
+
+                Type = NodeInputTypes.selectionInput,
+                Description = this.Description,
+                //TODO should this be the selected item to string or the index?
```

Comment:
in Dynamo these nodes usually return objects, and in XML they save the selected index as an int.

I'm not sure how / if these nodes `DSDropDownBase` currently work in JSON. @Racel did you find that drop down nodes worked in Revit with json?

I will implement this so that Value of Selection node serializes to the Item.Name  - but add a TODO to investigate.